### ISSUE TITLE
[Accessibility: ACC-264.1] Icon: UI type not conveyed and info read twice

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12603]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
UI elements type for all the UIs should be communicated to the user and all info must be conveyed only once to the user.
Providing all the information about the UI on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Icon](https://sap.github.io/fundamental-ngx/#/core/icon)
On accessing the ‘More info’ icon within the section ‘Communication’ the UI type ‘Button is not conveyed and information within the icon is conveyed twice to the user.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/cf55c957-10df-4c86-b3fb-2091a399b034">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
closing as there was no activity
